composer/algorithms/gradient_clipping/gradient_clipping.py

```diff
@@ -41,12 +41,16 @@ def apply_gradient_clipping(
             threshold by which if grad_norm / weight_norm is greater than this threshold then
             scale gradients by this threshold * (weight_norm / grad_norm) (for 'adaptive').
         fsdp_enabled (bool): Bool of if the model is a FSDP model or not.
+
+    Returns:
+        Union[torch.Tensor, None]: The total gradient norm before clipping for 'norm' clipping type,
```

Comment:
Let's just always return the result, not just for norm. 

It's a weird contract for a separate function downstream to know this behavior

Reply:
approved, not ideal, but makes sense that this is the best we can do